addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoBridgeHandler.java

```diff
@@ -311,4 +315,12 @@ public void webHookEvent(NAWebhookCameraEvent event) {
         }
     }
 
+    private String getWebHookURI() {
+        String webHookURI = null;
+        if (configuration.webHookUrl != null && configuration.readWelcome == Boolean.TRUE && webHookServlet != null) {
```

Comment:
A Boolean can be null. This is more a security check, I didn't check if in practice it can be null. It could depend on how ESH core is handling configuration parameters.

Reply:
I see that a default value is set for the config parameter so you're probably right and my null check is probably unneeded. I will do a quick test and then suppress it.

Reply:
I can confirm, after testing, the value is never null. So I followed your suggestion.

Reply:
looks good.